arm-recoveryservicesbackup/2016-06-01/swagger/recoveryservicesbackup.json

```diff
@@ -2561,6 +2738,17 @@
       },
       "discriminator": "objectType"
     },
+    "AzureBackupServerEngine": {
+      "description": "Backup engine type when Azure Backup Server is used to manage the backups.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BackupEngineBase"
+        }
+      ],
+      "properties": {},
```

Comment:
Hi, I've tried modifying the swashbuckle filters, but this doesn't seem to be supported. Looks like we can't make this change. Is this a blocker for sign-off?